office-365-feed/info.json

```diff
@@ -0,0 +1,189 @@
+  "description": "Ingest the IP, URLs used by Office 365 using the web service provided by Microsoft. The fetched indicators can be used to create a whitelist, blacklist etc. for your SIEM or firewall service and apply to its policy rules.",
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "category": "Threat Intelligence",
+  "icon_small_name": "office365_feed_32x32.png",
+  "icon_large_name": "office365_feed_80x80.png",
+  "ingestion_supported": true,
+  "ingestion_modes": [
+    "scheduled"
+  ],
+  "ingestion_preferences": {
+    "modules": ["threat_intel_feeds"]
+  },
+  "tags": [
+    "threatintel-inbound"
+  ],
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "title": "Regions",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "select",
+        "value": "Worldwide",
+        "options": ["China", "Germany", "USGovDoD","USGovGCCHigh","Worldwide"],
+        "description": "The regions from which to fetch indicators. Supports multi-select. For all regions, you need to select each region.\t",
+        "name": "regions"
+      },
+      {
+        "title": "Services",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "multiselect",
+        "value": "All",
+        "options": ["Common", "Exchange", "Sharepoint","Skype", "All"],
+        "description": "The services for which to fetch indicators. For all services, select the “All” option.",
+        "name": "services"
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True. ",
+        "value": true
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "fetch_indicator",
+      "title": "Fetch Indicators",
+      "description": "Fetch an IP Address and URL indicator(s) from the Office 365 feeds. The web service is a read-only API provided by Microsoft to expose the URLs and IPs used by Office 365. The fetched indicators can be used to create a whitelist, blacklist, EDL, etc. for your SIEM or firewall service to ingest and apply to its policy rules.",
+      "category": "investigation",
+      "annotation": "fetch_indicator",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Indicator Type",
+          "name": "indicator_type",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "select",
+          "options": [
+            "IPs",
+            "URLs",
+            "Both"
+          ],
+          "description": "Type of indicator that you want to fetch from Office 365 feeds.Valid types include: IPs, URLs and Both."
+        },
+        {
+          "title": "Confidence",
```

Comment:
Confidence, TLP, Expiry, Reputation, all can be removed from action input now. They are ingestion inputs